CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ All notable changes to this project will be documented in this file.
 - Made it possible to *not* return the `interp_basis_vector` array from beam
 interpolators.
 
+### Added
+- New keyword `check_kw` to `new_uvdata` that lets the user specify options to the
+checking of the created object.
+- New keyword `all_times_unique` to `calc_app_coords` that short-cuts the calculation
+when it is known that all input times are unique.
+
```

Comment:
This needs to be rebased and then this section needs to be combined with the Added section that's already in main, it's above the changed section.